Makefile

```diff
@@ -47,6 +47,10 @@ GC_FLAGS = $(BUILTIN_GC_FLAGS) $(EXTRA_GC_FLAGS)
 
 ASM_FLAGS ?= all=-trimpath="$$HOME"
 
+# TRIMPATH contains arguments to be passed to go to strip paths on Go 1.13 and
+# newer.
+TRIMPATH ?= $(shell [ "$$(go version | awk '{print $$3}' | sed -e 's/^.*\.//')" -ge 13 ] && echo -trimpath)
```

Comment:
I just realized there's a minor bug here; I neglected to consider the patch version (since Homebrew's version didn't have that), so I'll push up a fix with that in a second.  The internal pipeline will look like this:

```
go version | awk '{print $3}' | sed -e 's/^[^.]*\.//;s/\..*$//;'
```

which basically does the same thing, just picking out the part in the middle of `go1.13.1` (the 13) or the 13 in `go1.13`.

Reply:
Thanks for the explanation! I was thrown off by the `$$`, I didn't realize this was a requirement in Makefiles.

I've tested with both `go1.13` and `go1.13.1` and that new sed invocation seems to be working!

Reply:
Yeah, since Makefiles have variables introduced by `$` as well, you have to double it if you want to write a literal `$` so it can distinguish between its variables and the shell's..